src/cluster_legacy.c

```diff
@@ -2812,6 +2826,14 @@ void clusterUpdateSlotsConfigWith(clusterNode *sender, uint64_t senderConfigEpoc
                     dirty_slots_count++;
                 }
 
+                if (slotExportTargetName) {
+                    exporting_slots_count++;
+                }
```

Comment:
nit. the preference is using single lines in this case.
```suggestion
                if (slotExportTargetName)  exporting_slots_count++;
```

Reply:
Done. If these are hard style guidelines then should we add this to clang-format?

E.g. something like:

```
AllowShortIfStatementsOnASingleLine: All
AllowShortBlocksOnASingleLine: Always
```